### ISSUE TITLE
Enable 'Page' and 'Resource series' to appear on home page

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/bgs#344]
Add content type ‘page’ and ‘Resource series’ to the content that can be displayed in the ‘latest’ section on the homepage.

**Comment by angievanhegan** [comment]
@russ-manta Please can you add a cost/estimate to this? You were going to check it with Dilwar. Thanks very much

**Comment by russ-manta** [comment]
@angievanhegan we think 1 day. We are assuming that the pages and resource series would be added to the featured tab on home under latest - or would they need new tabs for either of those?

**Comment by angievanhegan** [comment]
@StephMSlack Can you look at Russ's question above? I'd assumed you wanted these to appear under featured, otherwise there's going to be a lot of tabs and it's veering towards an odd way of navigating the site rather than highlighting particular content. 

Please can you let me know if you'd like to go ahead with this one? Thanks

**Comment by StephMSlack** [comment]
@angievanhegan @russ-manta yes, this is to be able to add them to featured. we would not need new tabs for pages or for resource series. I need to check budgets before I can confirm go-ahead.

**Comment by angievanhegan** [comment]
@StephMSlack Any update on this?

**Comment by StephMSlack** [comment]
@angievanhegan can we leave this on the board and come back to it in November? At the moment it hasn't been a major issue and people have been finding a way to work around the restriction. I'd rather save the £ for the abstract system and make sure there is enough in the pot for that.

**Comment by angievanhegan** [comment]
@StephMSlack Absolutely.
